db/db.go

```diff
@@ -2,28 +2,37 @@ package db
 
```

Comment:
My problem with using session everywhere is that we'll constantly need to reference the specific database and and collection were using and we'll end up making those calls over and over again.

I had to change how we did this because the testing code was throwing an exception when it tried to close the session